src/main/java/io/github/thebusybiscuit/slimefun4/implementation/Slimefun.java

```diff
@@ -273,6 +280,17 @@ private void onPluginStart() {
             return;
         }
 
+        // Regex contains the following characters <,>,:,/,\,|,?,*,.
+        Pattern pattern = Pattern.compile("[\\.\\\\\\/\":><|?*]");
+
+        // Check if world contains an illegal character
+        for (World world : Bukkit.getServer().getWorlds()) {
+            if (pattern.matcher(world.getName()).find()) {
+                illegalWorld.add(world.getName());
+            return;
+            }
+        }
+
```

Comment:
Is that `return` indented properly?

Reply:
why are we even returning? That would stop this after the first match